test/uk/gov/hmrc/tai/connectors/BaseConnectorSpec.scala

```diff
@@ -116,26 +116,6 @@ class BaseConnectorSpec extends ConnectorBaseSpec {
       }
     }
 
-    "add extra headers for NPS" in {
```

Comment:
Why did we delete the verify headers test from this spec?

Reply:
I think we are verifying extra headers in the specific methods(in specific specs) where they are passed.

Reply:
Ahh, okay, can see the test in NpsConnectorSpec so all good